pymc3/math.py

```diff
@@ -14,6 +14,13 @@
 import numpy as np
 # pylint: enable=unused-import
 
+def tround(*args, **kwargs): 
```

Comment:
I don't want to clobber the builtin round anywhere it is imported. I guess the `t` will also remind us from time to time to check to see if it is needed anymore.